app/Helpers/render/achievement.php

```diff
@@ -70,7 +70,7 @@ function renderAchievementTitle(?string $title, bool $tags = true): string
         return '';
     }
     if (!Str::contains($title, '[m]')) {
-        return $title;
+        return htmlspecialchars($title, ENT_QUOTES, 'UTF-8');
```

Comment:
This breaks the display on the achievement details page:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/9b167a17-e843-41e1-a4d1-6301bd928a95)

![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/2f224f73-7cea-4b4a-8431-0f8043b190c6)